script/fetch_content.py

```diff
@@ -5,28 +5,45 @@
 
 
 def fetch_arguments():
+  """Fetch arguments from command line"""
+
   parser = argparse.ArgumentParser()
-  parser.add_argument('--url', help='add url')
-  parser.add_argument('--text', help='add text')
-  args = parser.parse_args() 
-  return args
+  parser.add_argument('url', help='add url')
+  parser.add_argument('text', help='add text')
+  args = parser.parse_args()  
+  return (args.url, args.text)
+
+
+def print_10_characters_before_and_after_text(text, r):
```

Comment:
Good job with the parameters.
One remark:
The parameters names could be better. Like
 * text
 * search_phrase

It could also be something like:
 * haystack
 * needle